numba/np/arraymath.py

```diff
@@ -1810,6 +1810,52 @@ def np_roll_impl(a, shift):
         return np_roll_impl
 
 
+@overload(np.tile)
+def np_tile(a, reps):
+    def np_tile_integer_impl(a, reps):
+        if reps < 0:
+            raise ValueError("tile(): second argument must be non-negative")
```

Comment:
"second argument" -> reps

Reply:
Looks like in vanilla NumPy, `tile` uses `repeat` under the hood, so passing a negative in `reps` results in `ValueError: negative dimensions are not allowed`. Should we match that?

Reply:
Yes, I think so